lesson8/src/one/Triangle.java

```diff
@@ -0,0 +1,17 @@
+package one;
+
+import one.Shape;
+
+public class Triangle implements Shape {
+
+    private final int a = 34;
+    private final int b = 56;
```

Comment:
I know it is hard to get the correct name for this kind of variable but you need to do better than these